actions/add-to-project/README.md

```diff
@@ -11,6 +11,8 @@ jobs:
         with:
           app-id: ${{ secrets.APP_ID }}
           app-pem: ${{ secrets.APP_PEM }}
+          project-url: project_board_url # optional: defaults to https://github.com/orgs/UCL-MIRSG/projects/3
```

Comment:
Can you instead make it like linting README https://github.com/UCL-MIRSG/.github/blob/046bf22a5c4da0f825eeabf13f8d6f02e798c71b/actions/linting/README.md?plain=1#L15-L27

Reply:
Sure, done.